packages/frontend/src/components/Stats/Stats.tsx

```diff
@@ -0,0 +1,22 @@
+import { useAllTimeStats } from 'src/hooks/useAllTimeStats';
+import { numberWithCommas } from 'src/utils/numberWithCommas';
+
+const Stats = () => {
+  const { isLoading, error, data } = useAllTimeStats();
+
+  if (isLoading) return <div className="text-xs invisible p-2">Loading all-time volume...</div>;
```

Comment:
Strange to have text that will never be shown?

Reply:
I wanted to prevent displaying error messages to users as it could undermine their confidence. Displaying a loading state might not be the best if we don't display an error state after. However, to avoid layout shift, we need some placeholder there. Invisible text is one approach.